dominion/frontend/src/component/Footer/Partners.js

```diff
@@ -1,15 +1,64 @@
 import React from 'react';
-import { Grid, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
 
-function Partners() {
+import { Grid } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+import bmf from '../../assets/images/logos/bmgf.png';
+import codeforafrica from '../../assets/images/logos/codeforafrica.png';
+import datazetu from '../../assets/images/logos/datazetu.png';
+import icfj from '../../assets/images/logos/icfj.png';
+import twaweza from '../../assets/images/logos/twaweza.png';
+
+const styles = () => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: '#2c2c2a',
+    padding: '3rem'
+  },
+  img: {
+    maxWidth: '100%',
+    height: 'auto'
+  }
+});
+
+function Partners({ classes }) {
   return (
-    <Grid container direction="column" justify="center" alignItems="center">
-      <Typography variant="h5">Dominion Setup</Typography>
-      <Grid style={{ paddingTop: '2rem' }}>
-        <Typography variant="h6">Partner&#39;s Logos</Typography>
+    <Grid>
+      <Grid
+        container
+        spacing={24}
+        direction="row"
+        justify="center"
+        alignItems="center"
+        className={classes.root}
+      >
+        <Grid item xs={2} style={{ padding: '3rem' }}>
+          <img
+            src={codeforafrica}
+            alt="Code for Africa"
+            className={classes.img}
+          />
+        </Grid>
+        <Grid item xs={2}>
+          <img src={datazetu} alt="Data Zetu" className={classes.img} />
+        </Grid>
+        <Grid item xs={2}>
+          <img src={twaweza} alt="Twaweza" className={classes.img} />
+        </Grid>
+        <Grid item xs={2}>
+          <img src={bmf} alt="Bill and Melinda Gates" className={classes.img} />
+        </Grid>
+        <Grid item xs={2}>
+          <img src={icfj} alt="Icfj" className={classes.img} />
+        </Grid>
```

Comment:
This design works well for large screens but for small/extra small screens, the partners logos become to small and disappear completely.

![screenshot from 2019-01-29 08-11-55](https://user-images.githubusercontent.com/1779590/51886112-7028b900-239f-11e9-838f-a889c6358e0b.png)